src/test/java/org/junit/rules/TemporaryFolderUsageTest.java

```diff
@@ -20,6 +20,7 @@
  */
 public class TemporaryFolderUsageTest {
 
+    public static final String ROOT_PATH = File.listRoots()[0].getAbsolutePath();
```

Comment:
Please make the a local variable in `newFolderWithPathStartingWithFileSeparatorThrowsIOException()`